index.bs

```diff
@@ -1128,35 +1130,6 @@ partial interface ModelBuilder {
         - *SumSquare*: Compute the sum of the square of all the input values along the axes.
 </div>
 
-### relu ### {#api-modelbuilder-relu}
-Calculate the [rectified linear](https://en.wikipedia.org/wiki/Rectifier_(neural_networks) function on the input tensor element-wise. The calculation follows the expression `max(0, x)`.
-<script type=idl>
-partial interface ModelBuilder {
-  Operand relu(Operand x);
-};
-</script>
-<div algorithm=relu>
-    **Arguments:**
-        - *x*: an {{Operand}}. The input tensor.
-
-    **Returns:** an {{Operand}}. The output tensor of the same shape as *x*.
-
-    Calculate the <a
-    href="https://en.wikipedia.org/wiki/Rectifier_(neural_networks)">rectified
-    linear function</a> on the input tensor element-wise. The calculation
-    follows the expression `max(0, x)`.
-
-    <div class="note">
-    The behavior of this operation can be generically emulated from the usage of
-    other operations as follow. However, user agents typically have a more
-    efficient implementation for it, therefore its usage is encouraged from the
-    performance standpoint.
-    <pre highlight="js">
-    return builder.max(builder.constant(0), x);
-    </pre>
-    </div>
```

Comment:
Do we need to keep the sample code that decomposes `relu`?

Reply:
Fixed.